ee/ucp/admin/configure/route-reflector-deployment.md

```diff
@@ -0,0 +1,210 @@
+           - name: etcd-certs
+             secret:
+               secretName: calico-etcd-secrets
+  ```
+  kubectl create -f calico-rr.yaml
+  ```
+  
+6) Use ```calicoctl``` to get the current bgp configuration and modify it to turn off node to node bgp mesh.
+
+   ```calicoctl get bgpconfig -o yaml > bgp.yaml```
+   
+   Set the nodeToNodeMeshEnabled to false in the spec.
+  
+   ``` bgp.yaml:```
+   ```
+   ....
+   spec:
+     asNumber: 63400
+     logSeverityScreen: Info
+     nodeToNodeMeshEnabled: false
+   ....
+   ```
+   Replace the bgp configuation with the modified ```bgp.yaml```
+   
+   ```
+   calicoctl replace -f bgp.yaml
+   ```
+   
+   If the bgpConfig object is empty, create the bgp config object.
+   ```
+   cat << EOF | calicoctl create -f -
+   apiVersion: projectcalico.org/v3
+   kind: BGPConfiguration
+   metadata:
+     name: default
+   spec:
+     logSeverityScreen: Info
+     nodeToNodeMeshEnabled: false
+     asNumber: 63400
+   EOF
+   ```
+   
+7) Create a bgp RR peer configuration. You can create multiple configuration for RRs
+
+   ```
+   calicoctl create -f - << EOF
+   apiVersion: projectcalico.org/v3
+   kind: BGPPeer
+   metadata:
+     name: bgppeer-global
+   spec:
+     peerIP: <IP_RR> #This will be the IP of node where the calico route reflector pod will be deployed.
+     asNumber: <AS_NUM> #Use the same asNumber from the configuration above step 6).
+   EOF
+   ```
+   The bgp RR configuration needs to be added for every Route Reflector that will be deployed.
+   Refer to https://docs.projectcalico.org/v3.1/usage/routereflector/calico-routereflector for more information.
+
+
+8) If you have calico-node pod running on the nodes marked for route reflector in step 3). You will need to manually delete the calico-node pod. We recommend this to avoid running both calico route reflector and calico-node pods on the same node.
+   
+   ```
+   kubectl get pods -n kube-system -o wide | grep ubuntu-0
+   calico-node-t4lwt                        2/2       Running   0          3h        172.31.20.89     ubuntu-0
+   
+   kubectl delete pod -n kube-system calico-node-t4lwt
+   ```
+  
+9) You can verify the calico-node from other nodes peering with the route reflector are status by downloading calicoctl        bundle on other nodes and checking for ```calicoctl node status```
+   ```
+   sudo ./calicoctl node status
+   Calico process is running.
+
+   IPv4 BGP status
+   +--------------+-----------+-------+----------+-------------+
+   | PEER ADDRESS | PEER TYPE | STATE |  SINCE   |    INFO     |
+   +--------------+-----------+-------+----------+-------------+
+   | 172.31.24.86 | global    | up    | 23:10:04 | Established |
+   +--------------+-----------+-------+----------+-------------+
+
+   IPv6 BGP status
+   No IPv6 peers found.
+   ```
+
+Please note that nodes marked for RR should not run any non system workloads. These nodes will not be gauranteed to work for ingress (swarm) or nodePort(kubernetes)
```

Comment:
Move this information to the beginning of the article. It will allow admins to make a more informed decision on how to roll this out.